pkg/csi/cinder/openstack/openstack_volumes.go

```diff
@@ -72,10 +72,10 @@ func (os *OpenStack) CreateVolume(name string, size int, vtype, availability str
 		return nil, err
 	}
 
-	// creating volumes from backups is available since 3.47 microversion
+	// creating volumes from backups and backups cross-az is available since 3.51 microversion
 	// https://docs.openstack.org/cinder/latest/contributor/api_microversion_history.html#id47
 	if !os.bsOpts.IgnoreVolumeMicroversion && sourceBackupID != "" {
-		blockstorageClient.Microversion = "3.47"
+		blockstorageClient.Microversion = "3.51"
```

Comment:
Feels like something that could work in 2 modes - volumes from backups in 3.47+ and volumes from backups and cross-AZ in 3.51+, but 3.51 is Rocky, so ancient anyway.

Reply:
yeah well, I was thinking that when this is new feature we could say that 3.51 is minimum version to get things working. The version before was Queens and its EOL was 2023-01-18. So I would say we are quite safe if we just use 3.51?

Reply:
Yes, that's my point, let's not bother too much about versions older than Rocky.